controller/src/main/java/org/jboss/as/controller/notification/NotificationSupports.java

```diff
@@ -84,7 +84,7 @@ public NotificationHandlerRegistration getNotificationRegistry() {
         private final ReadWriteLock lock = new ReentrantReadWriteLock();
 
         public NonBlockingNotificationSupport(NotificationHandlerRegistration registry, ExecutorService executor) {
-            this.registry = registry;
+                this.registry = registry;
```

Comment:
Nit: bad whitespace.

Actually this file isn't changed at all except for that.